tv_apps/smart-system/manifest.webapp

```diff
@@ -11,6 +11,7 @@
   "permissions": {
     "themeable":{},
     "alarms": {},
+    "system-app-only-audio-channels-in-app": {},
```

Comment:
We need the `system-app-only-audio-channels-in-app` permission to get System app's audio channel list form `navigator.mozAudioChannelManager.allowedAudioChannels`.